app/Http/routes.php

```diff
@@ -40,10 +40,10 @@
     ]);
 
     // Waiting rooms routes.
+    Route::model('waitingrooms', 'Gladiator\Models\WaitingRoom');
```

Comment:
I feel like these we're intentionally below the custom routes, do they still work?